src/stdatamodels/dqflags.py

```diff
@@ -21,6 +22,9 @@ def interpret_bit_flags(bit_flags, flip_bits=None, mnemonic_map=None):
     Wraps `astropy.nddata.bitmask.interpret_bit_flags`, allowing the
     bit mnemonics to be used in place of integers.
 
+    This method is deprecated and will be removed in a future version.
```

Comment:
Use this? https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated